### ISSUE TITLE
Not compatible with @ember/render-modifiers 2.0.0

### ISSUE THREAD

**Issue by oliverlj** [kaliber5/ember-bootstrap#1681]
Not compatible with @ember/render-modifiers 2.0.0

A lot of errors in test:
```
Error: Assertion Failed: You attempted to update `active` on `wrapperClass`, but it had already been used previously in the same computation.  Attempting to update a value after using it in a computation can cause logical errors, infinite revalidation bugs, and performance issues, and is not supported.
            
            `active` was first used:
            
            - While rendering:
              (instance of a `did-update` modifier)
            
            Stack trace for the update:
                at dirtyTagFor (http://localhost:7357/assets/vendor.js:55737:37)
                at setter (http://localhost:7357/assets/vendor.js:56023:7)
                at wrapperClass.set [as active] (http://localhost:7357/assets/vendor.js:14850:7)
                at wrapperClass.show (http://localhost:7357/assets/vendor.js:71097:19)
                at wrapperClass._onCollapsedChange (http://localhost:7357/assets/vendor.js:71188:14)
                at Object.updateModifier (http://localhost:7357/assets/vendor.js:67168:7)
                at CustomModifierManager.update (http://localhost:7357/assets/vendor.js:42020:18)
        message: >
            Promise rejected during "setting collapse to false expands this item": Assertion Failed: You attempted to update `active` on `wrapperClass`, but it had already been used previously in the same computation.  Attempting to update a value after using it in a computation can cause logical errors, infinite revalidation bugs, and performance issues, and is not supported.
```

https://discord.com/channels/480462759797063690/485923967148883998/906576631232098345

**Comment by simonihmig** [comment]
Yeah, thanks for reporting, I raised a PR almost the same time: #1680 😀

**Comment by jelhan** [comment]
I think this one could be closed as #1680 has been merged yesterday.

**Comment by jithine** [comment]
Is there a chance this fix will be pulled into 4.9.x ? I am on Ember 3, and ember-bootstrap 4.9.0.
Facing same issue with BsCollapse
